src/audiomixerboard.cpp

```diff
@@ -562,6 +605,13 @@ CAudioMixerBoard::CAudioMixerBoard ( QWidget* parent, Qt::WindowFlags ) :
 
 
     // Connections -------------------------------------------------------------
+
+    for (int i=0; i< static_cast<int>( vecpChanFader.size() ); i++) {
```

Comment:
Does this work with Qt4?

Reply:
Why not? This is not Qt specific, rather C++.  So it might depend on the compiler more than Qt version, but mist likely there is no problem. I did not try a build with Qt4 though.